build-coordinator/src/main/java/org/jboss/pnc/core/builder/BuildScheduler.java

```diff
@@ -29,5 +29,6 @@
  * @author <a href="mailto:matejonnet@gmail.com">Matej Lazar</a>
  */
 public interface BuildScheduler {
+    boolean hasBeenBuilt(BuildTask buildTask);
```

Comment:
Okey dokey, I'll prepare an update tomorrow. Maybe it would be a good idea to extract this checking-if-build-exists logic into a separate class? This way it would be very easy to find and extend it (if needed). I have a strong feeling that we will change it a couple of times in the future.